packages/plugin-react/src/index.ts

```diff
@@ -61,6 +61,8 @@ export type BabelOptions = Omit<
  * an `api.reactBabel` method.
  */
 export interface ReactBabelOptions extends BabelOptions {
+  ssr?: boolean
+  file: string
```

Comment:
I've added `file` and `ssr` properties to `ReactBabelOptions` object (which `api.reactBabel` hooks receive)